packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billingAccount.routing.js

```diff
@@ -100,6 +100,10 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href('telecom.telephony.billingAccount.billing.deposit', {
           billingAccount: billingAccountId,
         }),
+      serviceInformation: /* @ngInject */ (
+        billingAccountId,
+        tucVoipBillingAccount,
+      ) => tucVoipBillingAccount.fetchServiceInfo(billingAccountId),
```

Comment:
**question:** `.catch` is not needed here?

Reply:
I would tell that if we have error for service information, it would not be acceptable to access to this page without this information